libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -367,6 +367,15 @@ export interface BasicCard {
     title: string;
 }
 
+// @public
+export interface BotConfigAuth {
```

Comment:
Should we spell out the "config" abbreviations? Naming guidelines disfavor abbreviations: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

Reply:
@aosolis well, the documentation is for dotnet guidelines, and this is js. The [parity PR for dotnet](https://github.com/microsoft/botbuilder-dotnet/pull/6632/files#) has already been merged in, which follows the Teams botbuilder specs we have internally. @tracyboehrer Do you have input?

Reply:
Hi @aosolis, thanks for mentioning, the "config" naming refers to the invoke name "config/fetch" and "config/auth", and we used "config" in the platform code and the other sdk code base as well. So to reduce the confusion and keep the consistency, we probably want to use config as well here.